src/Native/Unix/System.Native/pal_memory.h

```diff
@@ -15,6 +15,6 @@ BEGIN_EXTERN_C
  *
  * Returns a pointer to the memory.
  */
-void* SystemNative_MemSet(void *s, int c, uintptr_t n);
+DLLEXPORT void* SystemNative_MemSet(void *s, int c, uintptr_t n);
```

Comment:
Were/are they using a .def file anywhere for these functions?
dllexport and .def files generate different names on NT/x86 and this is a reason .def files are preferred. But if there was no def file then nevermind.